lib/preview/httpsRedirectingServer.js

```diff
@@ -14,11 +15,11 @@ const createCert = async (domainsToProxy) => {
   const hosts = ['localhost', 'hslocal.net', ...additionalMkcertHosts];
   const { createCertificate } = await import('mkcert-cli');
   const { key, cert } = await silenceConsoleWhile(createCertificate, {
-    keyFilePath: `${__dirname}/key.pem`,
-    certFilePath: `${__dirname}/cert.pem`
+    keyFilePath: `${os.tmpdir()}/hstmp/hsLocalSshKey.pem`,
+    certFilePath: `${os.tmpdir()}/hstmp/hsLocalSshCert.pem`
   }, hosts);
-  unlinkSync(`${__dirname}/key.pem`);
-  unlinkSync(`${__dirname}/cert.pem`);
+  unlinkSync(`${os.tmpdir()}/hstmp/hsLocalSshKey.pem`);
+  unlinkSync(`${os.tmpdir()}/hstmp/hsLocalSshCert.pem`);
```

Comment:
Put these in the os temp dir feels a little cleaner pattern for the millisecond they exist before they get deleted. [Should be cross platform ](https://nodejs.org/api/os.html#ostmpdir)but I'll make sure